### ISSUE TITLE
Add ability to undo votes and/or clear all votes?

### ISSUE THREAD

**Issue by anishathalye** [anishathalye/gavel#22]
From his experience at HackPrinceton, @zacharyliu said that some issues that they had were:

* weird issue where one judge had 69 recorded votes within a few minutes, while the others had only ~20 votes by the end
* some judges started hitting random buttons even when we told them not to
* unable to delete bad votes from when judges hit random buttons before starting
* no way to undo accidental votes

Maybe we can improve this by adding:
* an ability to undo the last vote (to the judge UI)
* an ability for the organizers to clear votes for a specific judge / clear all votes

We need to have some discussion about exactly how this should be implemented.

**Comment by dargasia** [comment]
If votes that need to be undone are actually accidental (not like judges regretting their decision) we could also implement something similar to Gmail's Undo Send, i.e. holding the submission for a short amount of time before actually making the request.

**Comment by anishathalye** [comment]
True. Another way we could do it is by replaying all the decisions with the cancelled ones removed (this would be an O(n) operation, and for hackathon scale, probably fine).

**Comment by anishathalye** [comment]
Apparently people mis-click pretty frequently, so it would be nice to add the ability to undo votes.

**Comment by anishathalye** [comment]
Closing due to age/inactivity. I am no longer planning on doing this, though if someone submits a PR, I will review it.
